Azure/azure-cli
# [App Service] BREAKING CHANGE: `az functionapp`: Redact key value output on keys set

**Related command**
`az functionapp keys set`
`az functionapp function keys set`

**Description**
Currently, these two commands return the key value as part of the output. In this PR, we are redacting the key value to prevent accidental security leaks through CI logs. 

**Testing Guide**

1. Create functionapp `az functionapp create`
2. Add function app keys (`az functionapp keys set`) or function keys (`az functionapp function keys set`)
---

This checklist is used to make sure that common guidelines for a pull request are followed.
